docs/src/modules/components/ComponentLinkHeader.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react/jsx-no-target-blank */
+/* eslint-disable material-ui/no-hardcoded-labels */
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import GitHubIcon from '@material-ui/icons/GitHub';
+import SketchIcon from 'docs/src/modules/components/SketchIcon';
+import FigmaIcon from 'docs/src/modules/components/FigmaIcon';
+import BundleSizeIcon from 'docs/src/modules/components/BundleSizeIcon';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import W3CIcon from 'docs/src/modules/components/W3CIcon';
+import MaterialDesignIcon from 'docs/src/modules/components/MaterialDesignIcon';
+import { useSelector } from 'react-redux';
+import { makeStyles, fade } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: 0,
+    padding: 0,
+    listStyle: 'none',
+    display: 'flex',
+    flexWrap: 'wrap',
+    marginBottom: theme.spacing(2),
+    '& li': {
+      margin: theme.spacing(0.5),
+      padding: theme.spacing(0.5, 1, 0.5, 0.5),
+      border: `1px solid ${fade(theme.palette.action.active, 0.12)}`,
+      borderRadius: theme.shape.borderRadius,
+    },
+    '& a': {
+      color: theme.palette.text.primary,
+      textDecoration: 'none',
+      display: 'flex',
+      alignItems: 'center',
+    },
+    '& svg': {
+      marginRight: theme.spacing(1),
+    },
+  },
+}));
+
+export default function ComponentLinkHeader(props) {
+  const { headers } = props;
+  const classes = useStyles();
+  const t = useSelector((state) => state.options.t);
+
+  return (
+    <ul className={classes.root}>
+      <li>
+        <a
+          target="_blank"
+          rel="noopener nofollow"
+          href={`https://github.com/mui-org/material-ui/tree/next/packages/material-ui/src/${headers.components[0]}`}
```

Comment:
The source not part of the public API and a measure of last resort that is available without having to link it.

Reply:
I have moved this link to the end of the list. Were you suggesting to remove it? I don't know about you but I tend to go to the source git often, I assume it only help more advanced developers that know how to navigate the source.

Reply:
>  I don't know about you but I tend to go to the source quite often

You are not the target audience for the documentation. The better question is: What are you missing from the API docs that an ordinary user might also miss?

Reply:
> What are you missing from the API docs that an ordinary user might also miss?

The main value I see for the targeted audience is increasing trust, to allow them to audit the source before deciding to depend on the component.

Reply:
Auditing source is not one of the most important aspects of an introductory page of a component. API page is sufficient for that.

The use case you describe is also not the one you describe on the API page.

Reply:
Ok, removed